tests/setup.go

```diff
@@ -25,13 +25,13 @@ var (
 
 	dymensionImage = ibc.DockerImage{
 		Repository: DymensionMainRepo,
-		Version:    dymensionVersion,
+		Version:    "55d36971",
```

Comment:
using latest image leads to middlewareack error. We need to investigate that.
FYI @omritoptix